Test/exports/FIFO.dfy.py.check

```diff
@@ -0,0 +1,3 @@
+// https://github.com/dafny-lang/dafny/issues/4162
+// CHECK-L: assert "FIFO" == __name__
```

Comment:
What is CHECK-L ?

Reply:
https://github.com/stp/OutputCheck#check-l-string - like CHECK but looks for a "literal", exact match (aside from leading or trailing whitespace) rather than a regular expression.